boards/weact_f401ccu6/src/main.rs

```diff
@@ -143,11 +143,10 @@ unsafe fn setup_dma(
 /// Helper function called during bring-up that configures multiplexed I/O.
 unsafe fn set_pin_primary_functions(
     syscfg: &stm32f401cc::syscfg::Syscfg,
-    exti: &stm32f401cc::exti::Exti,
+    _exti: &stm32f401cc::exti::Exti,
```

Comment:
I think we should just delete this argument, this function is not called anywhere else.